rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -114,13 +114,13 @@ def _create_lports(self, lswitch, lport_create_args = [], lport_amount=1):
         network_cidr = lswitch.get("cidr", None)
         ip_addrs = None
         if network_cidr:
-            end_ip = network_cidr.ip + lport_amount
+            end_ip = network_cidr.ip + lport_amount + lport_ip_shift
             if not end_ip in network_cidr:
```

Comment:
Please correct me if I am wrong but I get the same result with network_cidr.first and network_cidr.ip (the first IP of the class, that is .0). Moreover I would add the possibility to skip a given set of IPs in the iprange since I would like to reuse _create_lports routine with datapath context from @jsitnicki and specify different IP ranges in multiple runs

Reply:
Sorry, please ignore this comment.

Reply:
Sounds good to me. iter_iprange() here is used as an iterator (L139), so I don't see performance issues.

Reply:
@hzhou8 ok, fine. What about lport_ip_shift? does it work for you?

Reply:
Yes, I think it should work. We can make it start from 1 by default, and provides the flexibility to specify other values for multiple runs.